src/doc/gui_manual/WorkingWithFiles/Read_Write_Options.rst

```diff
@@ -0,0 +1,184 @@
+.. _Read_Write_Options:
+
+Database Read and Write Options
+-------------------------------
+Several database plugins have options that effect reading and/or writing with that
```

Comment:
I believe the verb should be 'affect' not 'effect'. I have to look it up every time to double check. Here is a resource: https://www.dictionary.com/e/affect-vs-effect/